assets/js/modules/reader-revenue-manager/components/ReaderRevenueManagerSetupCTABanner.js

```diff
@@ -0,0 +1,198 @@
+				<Row>
+					<Cell size={ 12 }>
+						<Widget
+							noPadding
+							className="googlesitekit-setup-cta-banner googlesitekit-reader-revenue-manager-setup-cta-widget"
+						>
+							<Grid collapsed>
+								<Row>
+									<Cell
+										smSize={ 12 }
+										mdSize={ 8 }
+										lgSize={ 6 }
+										className="googlesitekit-setup-cta-banner__primary-cell"
+									>
+										<h4 className="googlesitekit-setup-cta-banner__title">
+											{ __(
+												'Grow your revenue and deepen reader engagement',
+												'google-site-kit'
+											) }
+										</h4>
+										<p className="googlesitekit-setup-cta-banner__description">
+											{ createInterpolateElement(
+												__(
+													'Turn casual visitors into loyal readers and earn more from your content with subscriptions, contributions, ad optimization and reader insight tools. <a>Learn more</a>',
+													'google-site-kit'
+												),
+												{
+													a: (
+														<Link
+															href={
+																readerRevenueManagerDocumentationURL
+															}
+															external
+															aria-label={ __(
+																'Learn more about reader revenue manager',
+																'google-site-kit'
+															) }
+														/>
+													),
+												}
+											) }
+										</p>
+
+										<div className="googlesitekit-setup-cta-banner__actions-wrapper">
+											<Button
+												className="googlesitekit-key-metrics-cta-button"
+												onClick={ onSetupCallback }
+											>
+												{ __(
+													'Set up Reader Revenue Manager',
+													'google-site-kit'
+												) }
+											</Button>
+											<Button
+												tertiary
+												onClick={ onDismiss }
+											>
+												{ __(
+													'Maybe later',
+													'google-site-kit'
+												) }
+											</Button>
+										</div>
+									</Cell>
+									{ ! isMobileBreakpoint &&
+										! isTabletBreakpoint && (
+											<Cell
+												alignBottom
+												className="googlesitekit-setup-cta-banner__svg-wrapper"
+												mdSize={ 8 }
+												lgSize={ 6 }
+											>
+												<SetupSVG />
+											</Cell>
+										) }
+									{ isTabletBreakpoint && (
+										<Cell
+											className="googlesitekit-setup-cta-banner__svg-wrapper"
+											mdSize={ 8 }
+										>
+											<SetupTabletSVG />
+										</Cell>
+									) }
+									{ isMobileBreakpoint && (
+										<Cell
+											alignBottom
+											className="googlesitekit-setup-cta-banner__svg-wrapper"
+											smSize={ 12 }
+										>
+											<SetupMobileSVG />
+										</Cell>
+									) }
+								</Row>
+							</Grid>
+						</Widget>
+					</Cell>
+				</Row>
+			</Grid>
+		</div>
+	);
+}
+
+ReaderRevenueManagerSetupCTABanner.propTypes = {
+	Widget: PropTypes.elementType.isRequired,
+	WidgetNull: PropTypes.elementType,
```

Comment:
Let's make it a required prop.
```suggestion
	WidgetNull: PropTypes.elementType.isRequired,
```